components/messaging/CurrentConversation.tsx

```diff
@@ -148,6 +168,9 @@ const CurrentConversation: React.FC = () => {
       <MessageForm
         user_id={currentConversation?.user_id}
         conversation_id={currentConversation?.conversation_id}
+        deletedList={deletedList}
+        setDeletedList={setDeletedList}
+        partner_has_deleted={currentConversation?.partner_has_deleted}
       />
```

Comment:
This component is rendered within `CurrentConversation` which will always have a `currentConversation` defined (otherwise we would not display the page at all).

So rather than making the props undefined, we can actually just match the props to the original type and assure typescript that the value will not be null.

Either add a `logical AND` at the start or a `non-null assertion operator` instead of that optional chained we have there